src/Modules/ScoreboardModule/Templates/Components/Row/Framemodel.mt

```diff
@@ -0,0 +1,244 @@
+    <property type="double" name="columnSpacing"/>
+    <property type="double" name="innerSpacing"/>
+    <property type="double" name="rowInnerHeight"/>
+    <property type="double" name="pointsWidth"/>
+    <property type="int" name="actionButtonCount"/>
+
+    <property type="double" name="textSize" default="2.4"/>
+    <property type="double" name="positionBoxWidth" default="9.6"/>
+
+    <template>
+        <framemodel id="player_row">
+            <!-- Scroll activation -->
+            <quad id="player_row_trigger"
+                  size="{{ w }} {{ rowHeight + rowSpacing }}"
+                  ScriptEvents="1"
+            />
+
+            <!-- Player Row Background -->
+            <PlayerRowBackground id="player_row_bg"
+                                 rowHeight="{{ rowHeight }}"
+                                 padding="{{ padding }}"
+                                 w="{{ w - positionBoxWidth }}"
+                                 h="{{ rowHeight }}"
+                                 x="{{ positionBoxWidth }}"
+            />
+
+            <!-- Position Box -->
+            <PositionBox width="{{ positionBoxWidth }}"
+                         height="{{ rowHeight }}"
+                         settings="{{ settings }}"
+            />
+
+            <!-- Custom Label Background -->
+            <CustomLabelBackground id="custom_gradient"
+                                   x="{{ w }}"
+                                   width="{{ w }}"
+                                   height="{{ rowHeight }}"
+            />
+
+            <frame pos="{{ positionBoxWidth + columnSpacing }} {{ rowHeight / -2.0 }}" z-index="10">
+                <!-- Flag -->
+                <Flag height="{{ rowInnerHeight }}" />
+
+                <frame pos="{{ rowInnerHeight * 1.5 + columnSpacing }} 0">
+                    <quad id="club_bg"
+                          size="{{ rowInnerHeight * 1.5 }} {{ rowInnerHeight * 0.75 }}"
+                          bgcolor="f00"
+                          valign="center"
+                          hidden="1"
+                    />
+                    <label id="club"
+                           class="text-primary"
+                           text="CLUB"
+                           pos="{{ rowInnerHeight * 0.75 }} 0"
+                           size="{{ rowInnerHeight * 2.0 }} {{ rowInnerHeight }}"
+                           valign="center2"
+                           halign="center"
+                           textsize="{{ Theme.UI_FontSize*2 }}"
+                           textcolor="{{ Theme.ScoreboardModule_Text_Color }}"
+                    />
+                </frame>
+
+                <!-- Player Name -->
+                <label id="name"
+                       pos="{{ rowInnerHeight * 3.0 + columnSpacing * 2.0 }} 0"
+                       size="{{ w / 3.0 }} {{ rowHeight }}"
+                       valign="center2"
+                       textsize="{{ Theme.UI_FontSize*2 }}"
```

Comment:
Use the `text-xl` utility class instead and for the others too.

Ref:
- `text-xs`: `Theme.UI_FontSize*0.8`
- `text-normal`: `Theme.UI_FontSize` (or `text-primary` for default colors + normal size)
- `text-md`: `Theme.UI_FontSize*1.2`
- `text-lg`: `Theme.UI_FontSize*1.5`
- `text-xl`: `Theme.UI_FontSize*2`
- `text-2xl`: `Theme.UI_FontSize*4`
- `text-3xl`: `Theme.UI_FontSize*6`
- `text-4xl`: `Theme.UI_FontSize*8`